distributed/cli/dask_submit.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+
+import click
+from tornado import gen
+
+from tornado.ioloop import IOLoop
+
+from distributed import rpc
+from distributed.cli.utils import check_python_3
+
+logger = logging.getLogger('distributed.dask_submit')
+
+
+@click.command()
+@click.argument('scheduler_address', type=str, required=True)
+@click.argument('filepath', type=str, required=True)
+def submit(scheduler_address, filepath):
+    @gen.coroutine
+    def f():
+        scheduler_host, remote_client_port = scheduler_address.split(':')
+        rc = rpc(ip=scheduler_host, port=remote_client_port)
+        remote_file = os.path.basename(filepath)
+        bytes_read = open(filepath, "rb").read()
+        yield rc.upload_file(filename=remote_file, bytes=bytes_read)
+        result = yield rc.execute(filename=remote_file)
+        if result['stdout']:
+            print(result['stdout'])
+        if result['stderr']:
+            print(result['stderr'])
```

Comment:
Perhaps we should write these to the `sys.stdout` and `sys.stderr` of this process?